src/content/ui-api/observability/prometheus/disable-monitoring/index.md

```diff
@@ -0,0 +1,80 @@
+---
+linkTitle: Disable Monitoring
+title: Disable Monitoring
+description: A guide explaining how to disable monitoring for a cluster
+weight: 50
+menu:
+  main:
+    identifier: uiapi-observability-clustermonitoring
+    parent: uiapi-observability-prometheus
+user_questions:
+  - How can I disable monitoring on my cluster ?
+aliases:
+  - /observability/prometheus/disable-monitoring
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-atlas
+last_review_date: 2022-11-21
+---
+
+In this document you will learn how you can disable monitoring for your cluster.
+
+## Introduction to monitoring
+
+Each cluster created on the Giant Swarm Platform benefits from our monitoring which allow us to provide you with 24/7 support to ensure best quality of service.
+
+Each cluster is monitored by a dedicated Prometheus instance.
+This comes by default and storage is reserved for data retention, storage size can be adjusted via [Prometheus Volume Sizing](https://docs.giantswarm.io/ui-api/observability/prometheus/volume-size/) feature.
+
+## Why would I like to disable monitoring?
+
+You might create clusters where you do not need support from our Operations team, for instance because you might just want to try something new for a short amount of time, or you are running a playground cluster where you know what you are doing.
```

Comment:
```suggestion
You might create clusters where you do not need support from our operations team, for instance because you might just want to try something new for a while, or you are running a playground cluster where you know what you are doing.
```